sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/AzureProxy.java

```diff
@@ -1,421 +0,0 @@
-        return createDefaultPipeline(swaggerInterface, new BearerTokenAuthenticationPolicy(credentials));
-    }
-
-    /**
-     * Create the default HttpPipeline.
-     *
-     * @param swaggerInterface The interface that the pipeline will use to generate a user-agent string.
-     * @param credentialsPolicy The credentials policy factory to use to apply authentication to the pipeline.
-     * @return the default HttpPipeline.
-     */
-    public static HttpPipeline createDefaultPipeline(Class<?> swaggerInterface, HttpPipelinePolicy credentialsPolicy) {
-        // Order in which policies applied will be the order in which they appear in the array
-        //
-        List<HttpPipelinePolicy> policies = new ArrayList<>();
-        policies.add(new UserAgentPolicy(getDefaultUserAgentString(swaggerInterface)));
-        policies.add(new RetryPolicy());
-        policies.add(new CookiePolicy());
-        if (credentialsPolicy != null) {
-            policies.add(credentialsPolicy);
-        }
-
-        return new HttpPipelineBuilder()
-            .policies(policies.toArray(new HttpPipelinePolicy[0]))
-            .build();
-    }
-
-    /**
-     * Create a proxy implementation of the provided Swagger interface.
-     *
-     * @param swaggerInterface The Swagger interface to provide a proxy implementation for.
-     * @param azureEnvironment The azure environment that the proxy implementation will target.
-     * @param httpPipeline The HTTP httpPipeline will be used to make REST calls. //* @param serializer The serializer
-     * that will be used to convert POJOs to and from request and response bodies.
-     * @param <A> The type of the Swagger interface.
-     * @return A proxy implementation of the provided Swagger interface.
-     */
-    @SuppressWarnings("unchecked")
-    public static <A> A create(Class<A> swaggerInterface, AzureEnvironment azureEnvironment,
-        HttpPipeline httpPipeline) {
-        String baseUrl = null;
-
-        if (azureEnvironment != null) {
-            final AzureHost azureHost = swaggerInterface.getAnnotation(AzureHost.class);
-            if (azureHost != null) {
-                baseUrl = azureEnvironment.url(azureHost.endpoint());
-            }
-        }
-
-        final SwaggerInterfaceParser interfaceParser =
-            new SwaggerInterfaceParser(swaggerInterface, createDefaultSerializer(), baseUrl);
-        final AzureProxy azureProxy = new AzureProxy(httpPipeline, interfaceParser);
-        return (A) Proxy
-            .newProxyInstance(swaggerInterface.getClassLoader(), new Class<?>[]{swaggerInterface}, azureProxy);
-    }
-
-    @Override
-    protected Object handleHttpResponse(final HttpRequest httpRequest, Mono<HttpDecodedResponse> asyncHttpResponse,
-        final SwaggerMethodParser methodParser, Type returnType, Context context) {
-        if (TypeUtil.isTypeOrSubTypeOf(returnType, Flux.class)) {
```

Comment:
Without this method, the client will lost the flexibility to access the original response body.

Reply:
My thought here was - The generated LRO methods in "private" service interface will be returning `Mono<ByteBuffer>`, so azure-core won't touch the body. The "LRO initialization Function" will be using this method and will have access to the raw-bytes.

Reply:
To add some context: the only reason we were overriding `handleHttpResponse` was to handle LRO return types. We no longer need it in the RestProxy level because (1). as mentioned in the previous comment we will return `Flux<ByteBuffer>` from service interface LRO proxy methods (2). then in the generated code, we will use `PollerFactory` to create and return `PollerFlux`.

Please let me know if you see an issue with this approach.